rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import vue from 'rollup-plugin-vue'
+import replace from "@rollup/plugin-replace";
+import commonjs from '@rollup/plugin-commonjs';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
+import pkg from './package.json';
+
+const commonPlugins = [
+  replace({
+    preventAssignment: true,
+    values: {
+      VERSION: JSON.stringify(pkg.version)
```

Comment:
Is it really needed?

Reply:
The version from package.json is already used in Vue.plugin. I wanna save this info for the package soon

Reply:
Then let's make it b more convenient way?
```
'import.meta.env.VERSION': JSON.stringify(pkg.version) 
```

(it's like modern alternative of `process.env`, https://dev.to/danawoodman/storing-environment-variables-in-sveltekit-2of3)